### DISCUSSION TITLE
OffCanvas Cart - Error message with Shortocode Button (Add To Cart)

### DISCUSSION THREAD

**Discussion by PGK37** [bootscore/bootscore#553] [Q&A]
When i use a shortcode button like this (in a landing page create specialy for a product) 
`[add_to_cart id="1422"  class="text-center" style=""]`

i have this error in the OffCanvas Cart on the right but the product is added to cart
![Capture d’écran 2023-08-22 à 19 19 09](https://github.com/bootscore/bootscore/assets/85485267/23fe49dd-265b-49a7-a8d4-2b8b67205808)

Any idea of this problem ?

**Comment by crftwrk** [comment]
It seems that you have set product to sold individually or have only 1 product in stock left and this one is already in your cart. To test, empty cart and add product twice. First success, second error. Check product settings.

For deeper investigation, please share a link to this site.

**Reply by PGK37** [reply]
Hello @crftwrk Basti

Thank you for the feedback. My product is not sold individually but the possibility of adding several.
The basket works well on the classic product single page (WooCoomerce page).

Can i share you the link in private ? The site is not in production…

Thank's a lot

**Reply by crftwrk** [reply]
you can send it to store(at)bootscore.me

**Reply by crftwrk** [reply]
You are trying to add a variation of a variable product direct via ajax to the cart. I think this won't work that way, because variable products needs selection first. You can try to inspect the button code in product page if selection is made and add HTML direct in your landing page.

But it seems to work fine except the error message. So I think simply hide the error by CSS in a Gutenberg HTML Block only on this page is the best option:


```html
<style>
  .cart-list .woocommerce-error {
    display: none;
  }
</style>
```

**Reply by PGK37** [reply]
Merci Basti ! 

I will try this solution I had thought of. ;-)
I just wanted to make sure that the error message is not a technical problem

**Reply by crftwrk** [reply]
You‘re welcome 👍
